docs/getting-started/continuous_integration.md

```diff
@@ -0,0 +1,112 @@
+is what your project is refered to in the [`projects`](https://github.com/google/oss-fuzz/tree/master/projects) directory of OSS-Fuzz.
+
+Your directory structure should look like the following:
+```
+project
+|___ .github
+|    |____ workflows
+|          |____ main.yml
+|___ other-files
+```
+
+main.yml for an example project:
+
+```
+name: CIFuzz
+on: [pull_request]
+jobs:
+ Fuzzing:
+   runs-on: ubuntu-latest
+   steps:
+   - name: Build Fuzzers
+     uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
+     with:
+       project-name: 'example'
+       dry-run: false
+   - name: Run Fuzzers
+     uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
+     with:
+       fuzz-time: 600
+       dry-run: false
+   - name: Upload Crash
+     uses: actions/upload-artifact@v1
+     if: failure()
+     with:
+       name: artifacts
+       path: ./out/artifacts
+```
+
+
+
+### Optional configuration
+
+`fuzz-time`: Determines how long CIFuzz spends fuzzing your project in seconds.
+The default is 600 seconds. The GitHub Actions max run time is 21600 seconds (6 hours).
+
+`dry-run`: Determines if CIFuzz surfaces errors. The default value is `false`. When set to `true`,
+CIFuzz will never report a failure even if it finds a crash in your project.
+This requires the user to manually check the logs for detected bugs. If dry run mode is desired,
+make sure to set the dry-run parameters in both the `Build Fuzzers` and `Run Fuzzers` action step.
+
+## Understanding results
+
+The results of CIFuzz can be found in two different places.
+
+* The run fuzzers log:
+    1. This log can be accessed in the `actions` tab of a CIFuzz integrated repo.
+    1. Click on the `CIFuzz` button in the workflow selector on the left hand side.
+    1. Click on the event triggered by your desired pull request.
+    1. Click the `Fuzzing` workflow.
+    1. Select the `Run Fuzzer` drop down. It should show the timestamps and results
+    from each of the fuzz targets.
+    ![Finding fuzzer output](https://raw.githubusercontent.com/google/oss-fuzz/master/docs/images/run_fuzzers.jpeg)
+
+
+*  The bug_report artifact:
+    1. When a crash is found by CIFuzz the Upload Artifact event is triggered.
+    1. This will cause a pop up in the left hand corner, allowing
+    you to download a zip file called `artifacts`.
+    1. `artifacts` contains two files:
+        * `test_case` - a test case that can be used to reproduce the bug.
+        * `bug_summary` - the stack trace and summary of the detected bug.
+    ![Finding uploaded artifacts](https://raw.githubusercontent.com/google/oss-fuzz/master/docs/images/artifacts.jpeg)
+
+
+## Feedback/Questions/Issues
+
+Create an issue in [OSS-Fuzz](https://github.com/google/oss-fuzz/issues/new) if you have feedback questions or issues with CIFuzz.
```

Comment:
"if you have questions of any other feedback"

Reply:
Done.